scripts/buildkite/release/generate-trigger.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
+COMMIT=$(buildkite-agent meta-data get "release-candidate-commit")
+VERSION=$(buildkite-agent meta-data get "release-version")
+BRANCH=$(buildkite-agent meta-data get "release-candidate-branch")
+CABAL=$(buildkite-agent meta-data get "release-cabal-version")
+
+cat << YAML
+steps:
+  - trigger: cardano-wallet
+    depends_on: add-release-commits
+    label: Trigger the main pipeline
+    build:
+        commit: $COMMIT
+        branch: $BRANCH
+        message: Release Candidate of $VERSION
+        env:
+            RELEASE_CANDIDATE: "$VERSION"
+            BUILDKITE_BRANCH: release-candidate-$VERSION
```

Comment:
```suggestion
            BUILDKITE_BRANCH: release-candidate/$VERSION
```

? 🤔 

I have the feeling that setting `BUILDKITE_BRANCH` to a name that is not actually a branch name will lead to trouble later.